src/components/call/FloatingCallWidget.scss

```
@@ -1,242 +1,246 @@
+	}
+
+	.call-video-area {
+		position: relative;
+		width: 100%;
+		flex: 1;
+		min-height: 380px;
+		background: var(
+			--m3-surface-container-high,
+			#e8e8e8
+		); // Light gray background
+		display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)